jupyterhub/tests/selenium/conftest.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from selenium import webdriver
+
+
+@pytest.fixture()
+def browser():
+    driver = webdriver.Firefox()
```

Comment:
Probably want to make sure we use headless firefox to avoid issues on CI:
```suggestion
    options = webdriver.FirefoxOptions()
    options.set_headless()
    driver = webdriver.Firefox(firefox_options=options)
```